openreview/tools.py

```diff
@@ -153,7 +153,7 @@ def get_profile(client, value, with_publications=False):
     return profile
 
 
-def get_profiles(client, ids_or_emails, with_publications=False, with_relations=False, as_dict=False):
+def get_profiles(client, ids_or_emails, with_publications=False, with_relations=False, with_preferred_emails=False, as_dict=False):
     '''
     Helper function that repeatedly queries for profiles, given IDs and emails.
     Useful for getting more Profiles than the server will return by default (1000)
```

Comment:
It would be nice if we indicated that `with_preferred_emails` takes an invitation id or at least set the default value to be `None` and change the logic on line 253 to `if with_preferred_emails is not None:`

Reply:
Done.